src/WorldState/WarframeStatusProvider.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace WorldState
+{
+    /// <summary>
+    /// A <see cref="WorldStateProvider"/> that uses online data from <a href="https://warframestat.us/">warframestat.us</a>.
```

Comment:
should probably specify https://api.warframestat.us, as there are several subdomains that do very different things.